RubeesEat.IntegrationTests/SeleniumTests/SplitBillSeleniumTests.cs

```diff
@@ -104,14 +104,17 @@ public void AddAndRemovePerson()
     {
         var page = Start<SplitBillPageObject>();
         
+        var selectionsStart = page.GetSelection();
+        
         var person = page.ClickAddPerson();
         
         Assert.That(page.GetPersonAmountAndNames().Length, Is.EqualTo(1));
+        Assert.That(page.GetSelection().Count < selectionsStart.Count());
         
         person.ClickRemoveButton();
         
         Assert.That(page.GetPersonAmountAndNames().Length, Is.EqualTo(0));
-
+        Assert.That(page.GetSelection().Count, Is.EqualTo(selectionsStart.Count()));
     }
 }
```

Comment:
I think we should assert if that specific person is removed from the options once we add it, and once we remove they are back on the options. So not only assert the length, but also if it was the right person